### Kuon kuon_control/SlewCmd

# Motor slew is the acceleration rate to meet the target velocity. Higher
# the slew, the harder the acceleration.
# Currently, slewing applies to all motors.

float32 slew    # power slew [0.0 - 1.0]
